.github/workflows/vib-redis.yaml

```diff
@@ -0,0 +1,75 @@
+name: 'vib'
+on: # rebuild any PRs and main branch changes
+  pull_request_target:
+    branches:
+      - master
+      - bitnami:master
+    types:
+      - assigned 
+      - opened 
+      - synchronize 
+      - reopened
+    paths:
+      - 'bitnami/redis/*'
+env:
+  CSP_API_URL: https://console.cloud.vmware.com
+  CSP_API_TOKEN: ${{ secrets.CSP_API_TOKEN }}
+  VIB_PUBLIC_URL: https://cp.bromelia.vmware.com
+jobs:
+  vib-validate: # make sure chart is linted/safe
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+      - uses: vmware-samples/vmware-image-builder-examples/vib-action@main
+
+  vib-verify-standalone: # verify in GKE
+    runs-on: ubuntu-latest
+    environment: vmware-image-builder
+    needs: [vib-validate]
+    name: Redis Standalone
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+      - uses: vmware-samples/vmware-image-builder-examples/vib-action@main
+        with: 
+          pipeline: redis/standalone/vib-platform-verify.json
+        env: 
+          TARGET_PLATFORM: 91d398a2-25c4-4cda-8732-75a3cfc179a1
+          
+  vib-verify-sentinel: # verify in AKS
+    runs-on: ubuntu-latest
+    environment: vmware-image-builder
+    needs: [vib-validate]
+    name: Redis Sentinel
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+      - uses: vmware-samples/vmware-image-builder-examples/vib-action@main
+        with: 
+          pipeline: redis/sentinel/vib-platform-verify.json
+        env: 
+          TARGET_PLATFORM: 7b13a7bb-011c-474f-ad71-8152fc321b9e   
+
+  vib-verify-replica: # verify in TKG
+    runs-on: ubuntu-latest
+    environment: vmware-image-builder
+    needs: [vib-validate]
+    name: Redis Replica
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+      - uses: vmware-samples/vmware-image-builder-examples/vib-action@main
+        with: 
+          pipeline: redis/replicas/vib-platform-verify.json
+        env: 
+          TARGET_PLATFORM: 7ddab896-2e4e-4d58-a501-f79897eba3a0           
+          
```

Comment:
Please address the 'no newline at end of file' warning. Recommendation: configure your IDE to do it for you:
- https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files